modules/CONT-02_Using-branches.yml

```diff
@@ -26,10 +26,10 @@ screens:
         - do: Go to another repo and then return to the class repo
           say: "However, a word of caution. When you leave the repository and come back, notice that GitHub automatically assumes you want to see the items on the master branch."
         - do: Click the branch dropdown and reselect the branch
-          say: "If you want to continue working on your branch, you will need to reselect it using the branch dropdown."
+          say: "If you want to continue working on your branch, you will need to reselect it using the branch dropdown. On the next page you will create your own branch."
```

Comment:
Hi @loranallensmith, here's the Certify glossary. We'll have to see what if any of that should be enshrined in the video content - ideally none of it if it wants to work in multiple formats.

- We have a curriculum - there will be one per partner. One for Git/GitHub, one for Docker, one for Chef, one for Trello
- Each curriculum has multiple courses. A course is a set of content covering a topic - something like GitHub for Everyone or GitHub for Developers.
- Events are specific instantiations of a course. We have an event for Intuit on 6/7 covering the GitHub for Developers course. Why not a class? Ruby doesn't like variables called class :smile: 
- A section is a set of content related to a broad topic - perhaps branching or even collaborating using PRs
- A module is a smaller set of content and is the unit of reusability. A section is comprised of multiple modules and a module has a set of screens relating to a discrete topic that may be reused in various courses.
- A screen is a particular video, lab, quiz, slide, etc displaying a small bite sized experience. Multiple screens build up a module.

Hope that's of some use?